circle.yml

```
@@ -1,17 +1,93 @@
-machine:
-  xcode:
-    version: 9.0
-  environment:
-    LANG: en_US.UTF-8
-dependencies:
-  override:
-    - bin/bootstrap-if-needed
-  cache_directories:
-    - "Carthage"
-test:
-  override:
-  - set -o pipefail
-  - xcodebuild test -workspace Action.xcworkspace -scheme Action -sdk iphonesimulator -destination "name=iPhone X" | xcpretty -c --test
-  - xcodebuild build  -workspace Action.xcworkspace -scheme Action-watchOS -sdk watchsimulator -destination "name=Apple Watch - 38mm" | xcpretty -c
-  - xcodebuild build -workspace Action.xcworkspace -scheme Action-macOS -sdk macosx -destination "arch=x86_64" | xcpretty -c
-  - xcodebuild build -workspace Action.xcworkspace -scheme Action-tvOS -sdk appletvsimulator -destination "name=Apple TV 1080p" | xcpretty -c
+---
+version: 2
+jobs:
+  "Action Tests":
+    working_directory: ~/RxSwiftCommunity/Action
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      XCODE_TEST_REPORTS: /tmp/xcode-test-results
+      LANG: en_US.UTF-8
+    macos:
+      xcode: '10.1'
```

This needs to be 
```suggestion
      xcode: '10.1.0'
```

See here @bobgodwinx :
https://discuss.circleci.com/t/xcode-10-1-image-released/26350